### PR TITLE
fix model height with header tools

### PR DESCRIPTION
**Pull request by AshleyBekemeier**
## Summary

- [Ticket](#)
- [Design](#)
- [Storybook](#)

### General

- [x] [Changeset added](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md)
- [x] The submitted code is organized and formatted according to our [💅 Style Guide](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-style-guide--docs).
  - [x] The code is formatted with Prettier.
  - [x] There are no linting errors.

### For new or updated components

https://swirl-storybook.flip-app.dev/?path=/docs/contributions-merge-publish--page

- [ ] The changes do not contain any breaking changes.
- [ ] The changes do not introduce new components that don't belong in the library (e.g. non-reusable components, highly specialized components, components including business logic)
- [ ] The component documentation is updated.
- [ ] The changes meet the [🤖 testing requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-testing--docs).
  - [ ] New features are tested.
  - [ ] In case of bug fixes, regression tests have been added.
  - [ ] All tests are 🟢.
- [ ] The changes meet the [🧏‍♀️ accessibility requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-accessibility--docs).
  - [ ] WCAG 2.1 Level A and Level AA requirements are met.
  - [ ] The Storybook a11y addon shows no errors.
  - [ ] The changes have been tested with a screen reader.
  - [ ] Keyboard controls have been tested, if applicable.
  - [ ] Components implementing form controls (inputs, buttons, selects, etc.) do not use Shadow DOM, and instead use Stencil's scoping mechanism
- [ ] The changes use our [🌈 theming concept](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-theming--docs).
  - [ ] Design tokens have been used where appropriate.
  - [ ] The component has been visually checked in combination with the "Light" and "Dark" theme.
- [ ] The changes meet our [🌍 internationalization requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-internationalization--docs).
  - [ ] No static text is used.
  - [ ] The component doesn't break with longer texts or different text wrappings.
  - [ ] Number, currency and date values can be formatted appropriately.
- [ ] The changes work in all supported browsers and viewports. See [📱 Responsive Design](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-responsive-design--docs)

### CODE REVIEW

File: packages/swirl-components/src/components/swirl-modal/swirl-modal.css [hunk 1]
```diff
@@ -382,7 +382,6 @@
 .modal__content {
   overflow-x: hidden;
   overflow-y: auto;
-  min-height: 100%;
```
Comment (Sqrrl) [inline packages/swirl-components/src/components/swirl-modal/swirl-modal.css:385]: I'm pretty sure this is needed for some use cases. To be safe, you could reset the min-height to 0 of `.modal__content` only within the `.modal--has-header-tools` variant.

Comment (AshleyBekemeier) [reply packages/swirl-components/src/components/swirl-modal/swirl-modal.css:385]: I added the height property back, does that fix those issues?

Comment (Sqrrl) [reply packages/swirl-components/src/components/swirl-modal/swirl-modal.css:385]: Not sure. Would have to test the edge cases. Is there anything to be considered against overwriting the min-height for the `modal--has-header-tools` variant only? I think that would be the safest approach with the least risk for regressions.

Comment (AshleyBekemeier) [reply packages/swirl-components/src/components/swirl-modal/swirl-modal.css:385]: It would work but it feels inconsistent to me. Do you remember what the edge cases were?

Comment (Sqrrl) [reply packages/swirl-components/src/components/swirl-modal/swirl-modal.css]: Nope. We can try without the min-height. Maybe you can make a quick smoke test with some Flip Web and Admin Console modals, once you've updated the Swirl version.

**Review by Sqrrl — APPROVED** [review]

**Comment by AshleyBekemeier** [comment]
@Sqrrl all the checks fail, is that relevant? i only changed one line

**Comment by Sqrrl** [comment]
> @Sqrrl all the checks fail, is that relevant? i only changed one line

For the Vercel checks you need to be invited to the Vercel team. Should be irrelevant here. You didn‘t change getflip.dev. The other checks should be green. Not sure why they aren‘t. Seems like your PR has no access to some secrets/tokens.  Could be because the PR was created before you were a member. I‘d say, try merging and see if the actions succeed against main.
